docker/config/webapp.env.template

```diff
@@ -3,7 +3,7 @@ DJANGO_DEV=True
 DJANGO_DEBUG=True
 DATABASE_URL=postgres://pontoon:asdf@postgresql/pontoon
 SESSION_COOKIE_SECURE=False
-SITE_URL=http://localhost:8000
+SITE_URL=http://#WEBAPP_HOST#
```

Comment:
I would make that variable contain the entire URL, schema included. In the future we might want to use those docker files for prod deploys, in which case we'll want to use HTTPS.